README.adoc

```diff
@@ -120,7 +120,7 @@ The ShrinkWrap test archive's contents are defined in this block, which is going
 
 Notice the `arquillian-managed.war` WAR name in the web archive. This is necessary if you don't want a randomly generated web archive name. This WAR name is also needed when the `inventory` service communicates with the `system` service to get the system properties.
 
-All of the classes in this service must be added to the web archive, otherwise, the code compiles successfully but fails at runtime when the injection of the testing resource class takes place. If you want to learn about the complex archive configuration, you can follow the http://arquillian.org/guides/shrinkwrap_introduction/[Arquillian guide].
```

Comment:
Why was the link to the shrinkwrap guide removed?

Reply:
Originally it was intended for user to learn more about how to define more complex archives. Now it's not necessary.

Reply:
I would still think that it's a good idea to provide them this link.